app/controllers/Api2.scala

```diff
@@ -162,16 +162,18 @@ class Api2 (override val stores: DataStores, conf: Configuration, override val a
 
   def getPlutoAtoms = APIAuthAction {  implicit req =>
 
-    val unprocessedAssetResponses: List[PlutoSyncMetadata] = stores.pluto.list()
+    val unprocessedAssetResponses: List[PlutoSyncMetadataMessage] = stores.pluto.list()
 
     val uploadsWithoutPlutoId = unprocessedAssetResponses.foldRight(Map[String, MediaAtom]())((upload, acc) => {
       if (!acc.contains(upload.atomId)) {
         previewDataStore.getAtom(upload.atomId) match {
           case Right(atom) => {
             val mediaAtom = MediaAtom.fromThrift(atom)
+
             mediaAtom.plutoData match {
+              case Some(plutoData) if plutoData.projectId.isEmpty => acc ++ Map(upload.atomId -> mediaAtom)
```

Comment:
Could you write 
```
case Some(plutoData) ifplutoData.projectId.exists => acc 
case _ => acc ++ Map(upload.atomId -> mediaAtom)
```
to simplify pattern matching?

Reply:
👍